### ISSUE TITLE
Graphical overview of tag distribution

### ISSUE THREAD

**Issue by almarklein** [almarklein/timetagger#125]
A graph (e.g. a donut) to give a visual clue how the activities are distributed in relation to each other. Maybe as part of the overview, or maybe as a visual "report".

Example:
![Example donut](https://play-lh.googleusercontent.com/DKGp81k0IWaFQcbP76uIvGSLatGJFs5jRhKJDHkBTQ76oUJeB4jPVoQAzrOcz4gGiNyw=w1738-h955-rw)

**Comment by dzmitry-lahoda** [comment]
good, but looks like it late product feature, not now:)

**Comment by almarklein** [comment]
It was a request by a user, and I like the idea. But not a high priority, indeed.

Some issues (like this one) I mark as "idea". Their direct advantage may not be very clear, or it may not be obvious how to integrate such a feature, it may need more discussion, or user input. Or for features that are just a lot of work.

**Comment by dzmitry-lahoda** [comment]
is user uses it day to day or once a week? in second export to some format can be solved by tableu or excel.  as per screenshot it is weekly.

**Comment by dzmitry-lahoda** [comment]
so it can not do visual, but make reporting out put(or data api?) to suite some consumer

**Comment by almarklein** [comment]
No, the idea is to have a more visual representation of how the amount of time for each tag relate. E.g. to see which tag was spent the most time on, and how much. You can see all that in the overview, but a more visual cue could be nice.

**Comment by dzmitry-lahoda** [comment]
i undrestand. so if that activity is once a week. not each hour? so can just do explore proper excel file and make consumer happy once a week.

**Comment by dzmitry-lahoda** [comment]
though experiment, imagine tt on watch. what features will remain in and what will not be still here? for me visualizing my time is not essential. that can be browser version with some standard widget.

**Comment by v-marinkov** [comment]
Here is a python library for creating such heatmaps: https://github.com/e-hulten/july which might prove useful for such a feature.
As I said in my https://github.com/almarklein/timetagger/issues/386#issuecomment-1930744336, tooltips on hover/tap would be essential to add.
